src/p11_child/p11_child_common.c

```diff
@@ -163,6 +171,7 @@ int main(int argc, const char *argv[])
     char *cert_b64 = NULL;
     bool wait_for_card = false;
     char *uri = NULL;
+	uint64_t old_chain_id = -1;
```

Comment:
While I understand the purpose, still setting unsigned var to `-1` looks a bit weird.

Reply:
This code is now gone in the latest update, so please disregard.